### PR TITLE
[viewer] fix(bi): onViewerAction

### PR DESCRIPTION
**Pull request by Barackos**

### CODE REVIEW

**Comment by Barackos** [comment]
@mharsat should be merged alongside your PR (#1536)

File: CHANGELOG.md [hunk 1]
```diff
@@ -66,6 +70,8 @@
   - [#1528](https://github.com/wix-incubator/rich-content/pull/1528) fixed `npm run watch` command for several packages
 - `storybook`
   - [#1533](https://github.com/wix-incubator/rich-content/pull/1533) ricos theme's palette examples are more realistic
+- `viewer`
```
Comment (mharsat) [inline CHANGELOG.md:73]: there is no change in viewer at all

**Review by mharsat — APPROVED** [review]
make sure to move your change to unreleased in changelog

File: CHANGELOG.md (hunk 1 above)
Comment (Barackos) [reply CHANGELOG.md:73]: oops forgot to remove it, thanks

**Pull request merged**